config/crd/bases/networking.cfargotunnel.com_clustertunnels.yaml

```diff
@@ -159,9 +204,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
Do you know why these were removed with your changes/kubebuilder version?

[The kubebuilder book](https://book-v1.book.kubebuilder.io/basics/status_subresource) seems to say it should be present when marked with `//+kubebuilder:subresource:status`

Reply:
I'm able to reproduce this lack of status with the proper (0.7.0) version of controller-tools as well. Could you try reproducing it?

Reply:
After even more testing, it seems like the version of kubebuilder is always v0.13.0 even if I use an older controller-gen version, seems to be my own setup. I'm a bit lost on how to solve this though.

Reply:
Will try and get back to you asap

Reply:
With controller-gen v0.7.0, it shows the status section, but v0.15.0 does not. I even created a new project as per the kubebuilder book and it does not match what they show. 

For now, could you use the v0.7.0 and run make manifests? I will try to update the whole setup to use the latest version of everything soon (TM) https://github.com/adyanth/cloudflare-operator/issues/99

Reply:
Now built with controller-gen v0.7.0! Addressed in [ca50b9d](https://github.com/adyanth/cloudflare-operator/pull/98/commits/ca50b9d61fe615aaf84099b880be5c530d3f9e4d)